browser/main/SideNav/SideNav.styl

```diff
@@ -18,7 +18,37 @@
   display flex
   text-align center
 
+.extra-buttons
+  position absolute
+  display flex
+  align-items center
+  justify-content flex-end
+  right 10px
+  top 24px
 
+.search
+  position relative
+  flex 1
+  display flex
+  max-height 0
+  overflow hidden
+  transition max-height .4s
+  margin -5px 10px 0
+  .search-input
```

Comment:
Can you change the background of the input according to the theme? For example, in Monokai theme here is the background of the search input:
![image](https://user-images.githubusercontent.com/12984316/51013107-44ca5f80-1593-11e9-9347-e1149010be69.png)

The background of other inputs:
![image](https://user-images.githubusercontent.com/12984316/51013124-5dd31080-1593-11e9-840a-559022dbdd03.png)